xc/device.ml

```diff
@@ -3010,9 +3014,9 @@ module Backend = struct
         let disks_cdrom, disks_floppy, disks_other =
           let partition (cdroms, floppies, other) ((_, _, media) as next) =
             match media with
-            | Dm_Common.Cdrom  -> (next :: cdroms, floppies, other)
-            | Dm_Common.Floppy -> (cdroms, next :: floppies, other)
-            | Dm_Common.Disk   -> (cdroms, floppies, next :: other)
+            | Dm_Common.Media.Cdrom  -> (next :: cdroms, floppies, other)
```

Comment:
You can add `let open Dm_Common.Media` just before the `match` if you want to keep the indentation level of the arrows